src/Eshopworld.Core/EswEventNameAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Eshopworld.Core
+{
+    /// <summary>
+    /// Describes an Esw Event Name Attribute
```

Comment:
You should describe the purpose of this attribute, so people know what its for + when to use it

Reply:
Comments updated - should be better now! 😄